[CI] XPackRestIT p0=indices.freeze/10_basic/Basic L74 Failure

```
java.lang.AssertionError: Failure at [indices.freeze/10_basic:74]: expected [2xx] status code but api [search] returned [503 Service Unavailable] [{"error":{"root_cause":[],"type":"search_phase_execution_exception","reason":"all shards failed","phase":"query","grouped":true,"failed_shards":[],"stack_trace":"Failed to execute phase [query], all shards failed\r\n\tat org.elasticsearch.action.search.AbstractSearchAsyncAction.onPhaseFailure(AbstractSearchAsyncAction.java:
...
```

https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.6+multijob-windows-compatibility/os=windows-2019/53/console
https://gradle-enterprise.elastic.co/s/t66es3e2pbe6g

Does not reproduce:
```
./gradlew ':x-pack:plugin:integTestRunner' --tests "org.elasticsearch.xpack.test.rest.XPackRestIT" \
  -Dtests.method="test {p0=indices.freeze/10_basic/Basic}" \
  -Dtests.seed=143749813EC5AF86 \
  -Dtests.security.manager=true \
  -Dtests.locale=mt-MT \
  -Dtests.timezone=Africa/Ouagadougou \
  -Dcompiler.java=13 \
  -Dtests.rest.blacklist=getting_started/10_monitor_cluster_health/*
```

--- a/x-pack/plugin/build.gradle
+++ b/x-pack/plugin/build.gradle
@@ -99,6 +99,8 @@ testClusters.integTest {
   setting 'xpack.security.transport.ssl.verification_mode', 'certificate'
   setting 'xpack.security.audit.enabled', 'true'
   setting 'xpack.license.self_generated.type', 'trial'
+  // disable ILM history, since it disturbs tests using _all
+  setting 'indices.lifecycle.history_index_enabled', 'false'
   keystore 'bootstrap.password', 'x-pack-test-password'
   keystore 'xpack.security.transport.ssl.secure_key_passphrase', 'testnode'
 

